src/libcore/platforms/react-native.js

```diff
@@ -0,0 +1,211 @@
+// @flow
+
+import { NotEnoughBalance, NetworkDown } from "@ledgerhq/errors/lib";
+import type { Core, CoreStatics } from "../types";
+import { reflect } from "../types";
+import { setLoadCoreImplementation } from "../access";
+import { setRemapLibcoreErrorsImplementation } from "../errors";
+
+export default (arg: { getNativeModule: (id: string) => any }) => {
+  const { getNativeModule } = arg;
+
+  async function loadCore(): Promise<Core> {
+    // Abstract out the libcore interface into a higher level API.
+
+    const mappings: Object = {};
+
+    const wrappers = {
+      // FIXME need to fix it in RN bindings
+      hex: value => value.replace(/[< >]/g, "")
+    };
+
+    function wrap(id, ref) {
+      if (!ref) return ref;
```

Comment:
how can you/why do you return `ref` is there is no `ref` ?

Reply:
some method returns optional results. For instance `CoreOperation#getFees(): ?CoreAmount;`

in that case if it's a `null` we need to return a `null`

if it's a `ref` (like a `{ id: "uuid of a CoreAmount" }`) it needs to wrap it by a `new CoreAmount(ref)`

Reply:
👍